### PR TITLE
Add browser not supported message to video player.

### PR DESCRIPTION
**Pull request by phamann**
This add's a browser support table to the video player and displays an error message prompting the user to update their browser if it is not supported.

Unfortunately i've had to do some hacky UA detection to achieve this, please don't judge me :crying_cat_face: 

/cc @gklopper @rich-nguyen

### CODE REVIEW

File: static/src/javascripts/bootstraps/media.js [hunk 1]
```diff
@@ -40,6 +41,7 @@ define([
     history,
     images,
     techOrder,
+    checkBrowserSupport,
```
Comment (rich-nguyen) [inline static/src/javascripts/bootstraps/media.js]: `supportedBrowsers`

**Comment by rich-nguyen** [comment]
:+1:

**Comment by gklopper** [comment]
:hammer:

**Comment by gklopper** [comment]
Lets see that switch and Safari support then :+1:

**Comment by phamann** [comment]
cc @gklopper have added switch and safari

**Pull request merged**